ONSdigital/rm-actionexporter-service
# Downgrade to spring boot 1.5

# Motivation and Context
We've been unable to get SFTP upload of print files (implemented with Spring Integration Framework) working in the new Spring Boot 2.x upgrade, so until this is migrated out to the new `ras-rm-print-file`, we have to temporarily revert Spring Boot from latest 2.2.x to latest 1.5.x.


# What has changed
- Downgraded Spring Boot from 2.2.9 to 1.5.22.
- restored old lib dependencies, along with old config in YAML, and removed new SB2 class SecurityConfig.java.
- left slf4j logging in place




# How to test?
- `mvn clean install`
- `mvn spring-boot:run`
- deploy action exporter with new image to a sandbox cluster
- set bucket in sandbox to a test bucket using ras-rm-cli
- run acceptance tests on sandbox cluster 
- use port forwarding to connect to Response Ops on localhost:8085, and create a new collection exercise for same day
- check that the print file is in the test bucket 





# Links
- https://collaborate2.ons.gov.uk/jira/browse/RAS-82
